app/models/user.rb

```diff
@@ -49,4 +50,8 @@ def unique_code_for_digest
   def self.anonymous
     User.find_or_create_by(gravity_user_id: 'anonymous')
   end
+
+  def contact_information?
+    !(self[:name].blank? || self[:email].blank? || self[:phone].blank?)
```

Comment:
Just a suggestion: could replace this with something like this:
```suggestion
    self[:name].present? && self[:email].present? && self[:phone].present?
```
This negation combined with unless on top is a bit confusing😆